packages/vaadin-themable-mixin/test/lit-setup.js

```diff
@@ -0,0 +1,31 @@
+import { LitElement, unsafeCSS } from 'lit';
+import { ThemableMixin } from '../vaadin-themable-mixin.js';
+
+/**
+ * This is used for overriding the function that defines custom elements in the test suites.
+ * By default, the suite creates PolymerElement based custom elements, but in the
+ * -lit.test.js tests, we specifically want to create LitElement based custom elements instead.
+ */
+window.defineCustomElementFunction = (name, parentName, content = '', styles) => {
+  class CustomElement extends ThemableMixin(parentName ? customElements.get(parentName) : LitElement) {
```

Comment:
```suggestion
  const parentElement = parentName ? customElements.get(parentName) : LitElement;
  class CustomElement extends ThemableMixin(parentElement) {
```

Consider extracting the parent element into a variable to improve readability.

Reply:
Done.